examples/website-demos/social-messaging/src/components/ChatContainer/ChatContainer.tsx

```diff
@@ -1,14 +1,16 @@
 import { useEffect, useState } from 'react';
 
-import { ChannelSort, LiteralStringForUnion, StreamChat } from 'stream-chat';
+import type { Channel as TypeChannel } from 'stream-chat';
```

Comment:
just for consistency, in the other app we've been using the name `StreamChannel` when we need to reassign the type